### PR TITLE
refactor: remove old demos and update readme

### LINKED ISSUE #108: Remove old demos
### Is your feature request related to a problem? Please describe.

N/A

### Describe the solution you'd like

Remove at_cookbook, at_hello_world

@gkc should we remove at_demo_data?

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by HamdaanAliQuatil**
Closes #108 

**- What I did**

Remove old demos and update the readme accordingly

**- How I did it**

- [x] Remove at_cookbook and at_hello_world demos.
- [x] Updated README to remove at_cookbook, at_hello_world demos and add sshnp_docker_demo.

**- Description for the changelog**

remove old demos and update readme

### CODE REVIEW

**Comment by JeremyTubongbanua** [comment]
also have new notifications demo #111

**Comment by JeremyTubongbanua** [comment]
Thank you, @HamdaanAliQuatil :)

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by JeremyTubongbanua — APPROVED** [review]

**Comment by xavierchanth** [comment]
Thanks for doing this @HamdaanAliQuatil

**Review by xavierchanth — DISMISSED** [review]

**Review by JeremyTubongbanua — DISMISSED** [review]

File: README.md
Comment (TylerTrott) [inline README.md:1]: Typo: `sshnp_docker_demo`: A demo of to establish an

Should be: `sshnp_docker_demo`: A demo to establish an

Comment (TylerTrott) [reply README.md:1]: Typo: `at_notifications`: A demo of using `.notify` and `.subscribe`

Should be: `at_notifications`: A demo using `.notify` and `.subscribe`

File: at_notifications/README.md
Comment (TylerTrott) [inline at_notifications/README.md:1]: Typo: This is a demo of using `.notify` and `.subscribe` 

Should be: This is a demo using `.notify` and `.subscribe`

Comment (TylerTrott) [reply at_notifications/README.md:1]: Typo: `subscribe-to-notifier.dart` @subscriber is listens for notifications received from @notifier

Should be: `subscribe-to-notifier.dart` @subscriber listens for notifications received from @notifier

Comment (TylerTrott) [reply at_notifications/README.md:1]: Typo:  To run this demo, you will need 2 atSigns and thier .atKeys in `~/.atsign/keys` directory.

Should be:  To run this demo, you will need 2 atSigns and their .atKeys in `~/.atsign/keys` directory.

**Review by TylerTrott — CHANGES_REQUESTED** [review]
Few simple typos

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by JeremyTubongbanua — APPROVED** [review]

**Review by TylerTrott — APPROVED** [review]
Lgtm

**Pull request merged**